how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -166,19 +188,19 @@ $ sudo -E kubectl get pods --all-namespaces | grep dns | grep Running && echo "O
 
 ### Allow run pods in master node
 
-By default, the cluster will not schedule pods in the master node to allow that run:
+By default, the cluster will not schedule pods in the master node. To enable master node scheduling, run:
 
 ```bash
-# allow master node run pods
+# allow master node to run pods
 $ sudo -E kubectl taint nodes --all node-role.kubernetes.io/master-
 ```
 
 
 ### Create a unstrusted pod using Kata Containers
 
 By default, all pods are created with the default runtime configured in CRI containerd plugin.
-If a pod has the `io.kubernetes.cri.untrusted-workload annotation` set as
-`"true"`, the CRI plugin will run the pod with the Kata Containers runtime.
+If a pod has the `io.kubernetes.cri.untrusted-workload` annotation set to
+`"true"`, the CRI plugin will run the pod with the [Kata Containers runtime](https://github.com/kata-containers/runtime).
```

Comment:
Suggested rewrite to stay active: `"true"`, the CRI plugin runs the pod with the [Kata Containers runtime](https://github.com/kata-containers/runtime).